docs/prio-processor-guide.md

```diff
@@ -0,0 +1,321 @@
+# Prio Processor
+
+`mozilla/prio-processor` is a container application that implements the privacy
+and correctness guarantees of Prio, a privacy-preserving aggregation system. The
+processor inter-operates with the Firefox Data Platform by an agreed convention
+of data exchange across cloud storage.
+
+The initial release (v1.0) contain an automated workflow for batched processing
+of ["prio"
+pings](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/prio-ping.html)
+that are ingested via
+[mozilla/gcp-ingestion](https://github.com/mozilla/gcp-ingestion).
+
+This processor aggregates [Origin Telemetry
+pings](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/origin.html)
+configured to measure blocklist exceptions in Firefox on pre-release channels.
+
+## Quick start
+
+```bash
+# build the container
+make build
+
+# run the container on localhost
+make test
+```
+
+Set the following environment variables in a `.env` file in the current
+directory:
+
+```bash
+GOOGLE_APPLICATION_CREDENTIALS_A=...
+GOOGLE_APPLICATION_CREDENTIALS_B=...
+GOOGLE_APPLICATION_CREDENTIALS_ADMIN=...
+```
+
+To simplify testing, resources for each container may be co-located in the same
+project. In this case, all three application credential variables may point to
+the same service account.
+
+The integration tests are currently configured for `prio-a-nonprod`,
+`prio-b-nonprod`, and `prio-admin-nonprod` under Google Cloud Platform. To
+request service account access, file a bug under [Data Platform and Tools ::
```

Comment:
We should have docs on service accounts specifically which include detailed instructions on what to request (I actually filed an issue: https://github.com/mozilla/firefox-data-docs/issues/370). We have this but it's bigquery-specific:

https://docs.telemetry.mozilla.org/cookbooks/bigquery.html#gcp-bigquery-api-access

Once that's done we should have this link to that. No need to block this on it, more just me making a note about it: